README.md

```diff
@@ -68,7 +68,24 @@ Clear containers:
 docker rm -f $(docker ps -a -q)
 ```
 
-## Deploying to Google Cloud Platform
+### Deploying to Google Cloud Platform
 ```
 gcloud app deploy app.yaml --project <your-project-id>
-```
\ No newline at end of file
+```
+
+## Rendering
+Once you have the service up and running, you'll need to implement the differential serving
+layer. This checks the user agent to determine whether prerendering is required.
+
+### Web components
+If you are using web components v0 (deprecated), you will need to enable Shady DOM to
+render correctly. In Polymer 1.x, which uses web components v0, Shady DOM is enabled by default.
+If you are using Shadow DOM, override this by setting `?dom=shady` when directing requests
+to the bot-render service.
+
+If you are using web components v1, simply pass `wc=1` when directing requests to the bot-render
```

Comment:
Although for V1 the `webcomponents-loader.js` case is preferred, users could still be loading `webcomponents-lite.js` (which bundles all polyfills and supports query param-based forcing).  So there'd be two cases:

* User loads no polyfill or `webcomponents-loader.js`, in which case they should flag to the renderer service to inject shadydom and force it on (currently what you have `wc=1` doing)
* User loads `webcomponents-lite.js`, in which case they should just set `wc-shadydom=true` to force shadydom on

Maybe just document the other case here, and possibly consider making the "inject shadydom" flag more explicit, like `wc-inject-shadydom=true`?